util/internal/integrationtest/common/src/main/java/org/hibernate/search/v6poc/util/impl/integrationtest/common/rule/ExpectedLog4jLog.java

```diff
@@ -0,0 +1,239 @@
+/*
```

Comment:
I think the commit message is missing `rule` after JUnit?

Reply:
Right. Done.